.gitignore

```diff
@@ -7,4 +7,6 @@ coverage
 .npmrc
 
 lerna-debug.log
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+
+**/.DS_Store
```

Comment:
This change isn't connected to the typescript stuff; I've just seen these files popping up locally, and figured we should add them to the gitignore